static/src/javascripts/projects/commercial/modules/prebid/price-config.js

```diff
@@ -1,6 +1,13 @@
 // @flow strict
 
-import type { PrebidPriceGranularity } from 'commercial/modules/prebid/types';
+export type PrebidPriceGranularity = {
```

Comment:
This probably doesn't need to be an export type since we do not rely on the type definition anywhere else in the codebase... 🤔